### ISSUE TITLE
[Editor] House number, postal (zip) code and number of floors do not have icons

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#6933]
It looks weird, especially on iOS: when values are already there, it is not clear at all what these fields mean/contain.

Are there any ideas about good icons? What other apps are using?

iOS | Android | Android
----|---------|--------
![telegram-cloud-photo-size-2-5203994710883029981-y](https://github.com/organicmaps/organicmaps/assets/170263/ffdc0d91-75f1-4894-b354-64e22654b001)|![telegram-cloud-photo-size-2-5203994710883029978-y](https://github.com/organicmaps/organicmaps/assets/170263/5a870412-c070-4bfe-9b55-6c0a6809e8b9)|![telegram-cloud-photo-size-2-5203994710883029979-y](https://github.com/organicmaps/organicmaps/assets/170263/99cdf1a8-b2db-4f7d-bb59-b9f67be074f6)

**Comment by zyphlar** [comment]
I wouldn't add icons for each sub-field of address, I would just rearrange the fields to leave less blank space. Also for en-US housenumber should be first.

So for example the icon could go on the first line and then subsequent lines could have less left-margin so it doesn't feel like something's missing.

**Comment by 11happy** [comment]
I also agree with @zyphlar like without icons it would be more clear, they should be just entry fields like this.Example (taken from amazon address input fields)
![image](https://github.com/organicmaps/organicmaps/assets/76656712/52255787-a2b4-4d4c-86bc-611ca0a688a3)

**Comment by zyphlar** [comment]
Idk if it's possible to put the 📍marker next to the word Address on iOS/Android but in general it's less critical than with Details which are a large list of parallel items recognized by logo (like social network) and could probably just be removed for that one item (the pin doesn't add that much understanding to the word Address, the main potential for confusion or frustration will be the fields being ordered differently from local postal customs and the unneeded whitespace.

**Comment by Misalf-git** [comment]
Throwing in, I think it's good enough on Android where descriprive text is used / without icons.

Adopting that to iOS may possibly be good as well and enough to understand the meaning of already filled entries.

I can't come up with ideas for icons describing things like ZIP code, State, etc. (@zyphlar's list above) in a familiar form and without requiring extra learning.
Those definitions are rather abstract and not easy to visualize distinctly.

**Comment by Meet734** [comment]
I believe that incorporating icons will simplify user interaction and enhance the overall user experience. My suggested icons for the following fields are:

- Street : 
![Screenshot_20240121_062504](https://github.com/organicmaps/organicmaps/assets/95461453/028d067d-85f9-4692-9115-a8666306517f)
or
![image](https://github.com/organicmaps/organicmaps/assets/95461453/4c01bc63-eb94-4b7f-bd3c-b0a5ff937553)
 
- Building No : 
![image](https://github.com/organicmaps/organicmaps/assets/95461453/fb64a9f7-64b6-49e1-ba56-4cd4fd92eb9a)
 
- Floor No : 
![image](https://github.com/organicmaps/organicmaps/assets/95461453/8dfb01d9-79bd-4d50-8fee-6b1dab5a9828)
 
- Zip code : 
![image](https://github.com/organicmaps/organicmaps/assets/95461453/55b6f142-5ed5-442d-85d7-9b5a5e4b29c9) 

I found these icons on [https://fontawesome.com/](url)

**Comment by zyphlar** [comment]
Those can be fine icons but the Floor icon would be interpreted as `addr:unit` in America and post/zip/city are still difficult. I think branded icons are fine for contact methods but people are pretty used to thinking of addresses as they appear on envelopes. The issue is just the layout and wording since each country handles that differently.

**Comment by slackuj** [comment]
Hello,
I recently undertook a learning path in Qt viz. "Esri x Qt: Exploring with ArcGIS Maps SDK for Qt".

Esri the world's leading supplier of geographic information system (GIS) software, web GIS, and geodatabase management applications has its own icon-library.I came to know about the calcite-UI icons.
https://github.com/Esri/calcite-ui-icons
https://developers.arcgis.com/calcite-design-system/icons/?library=Calcite%20UI

What about using the icons from such libraries!?

Moreover, what about constructing a collection of icons (Organic Maps's Icon Library---OMI Library) used in Organic Maps?I would heavily like to suggest to construct a dedicated library for icons and allowing everyone to contribute to that library!!!

**Comment by matheusgomesms** [comment]
Reopening this issue because #10208 only tackles Android, not iOS.
